diagnostic.md

```diff
@@ -5,74 +5,82 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+To work with a server that stores info so that we can have functionality such as user IDs, game stats, two users able to use app at once and the information will persist.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+M - model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+C - controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Views are for non-SPA (single page application) apps...which is out dated technology.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create, Read, Update and Destroy.
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+routes
```

Comment:
Technically the controller, but the router does select the appropriate controller.